client/login/wp-login/components/one-login-layout.tsx

```diff
@@ -72,6 +74,7 @@ const OneLoginLayout = ( {
 				/>
 			}
 			verticalAlign="center"
+			wrapperClassName="wp-login__one-login-layout-content-wrapper"
```

Comment:
The premise of the wireframes is that they're predictable across usages and we don't want to add overrides. Allowing to pass an arbitrary class name makes that premise moot.

What do we want to achieve here? Why can't we keep the default `gap`?

Reply:
I want to reduce the gap, yes. See description.

Reply:
Then my suggestion here is to add a prop to the content wrapper so we're explicit about it. We can provide different gap options if that's what we need. It just needs to be explicit, and we need a designer's approval as well. @fditrapani @fcoveram.

Reply:
Spunds good. Can you work out the Step Container changes for accepting these gap prop? It sounds like you own that part, so it's safer if you can extend and adjust the container to suit needs. Here, we need a smaller gap, it's in the DS Pass.

We are also in a bit of rush to get things moving forward. Unless no issues in testing, I intend to merge this before EOD tomorrow (Thursday) 🤞

Reply:
This is what I'm trying to accomplish via reducing the gap i.e. the gap between heading & content is unnaturally wide here (and I think adding the same padding/gap below will make it worse):

<img width="500" alt="Screenshot 2025-07-04 at 8 50 14 PM" src="https://github.com/user-attachments/assets/d513c9bd-8893-4fe4-b544-80a163c85fdf" />

There are several issues that I'm seeing right now, also noted above. I am going to switch the heading out of the wireframe as it's neither portable to the multiple sub-headings we are seeking for some of the OAuth client screens. That should remove the need for a wrapper class.

Reply:
And to be very clear, I think you're right in reducing the gap, as pointed out by Francisco as well. I just want to do it in a way that's maintainable and don't bring further inconsistencies.

Stakeholders (p1744733006498779/1744293960.380589-slack-C06FCT6EEHH) loved the onboarding consistency project and our job is to make sure they'll keep loving it.

Reply:
>We are also in a bit of rush to get things moving forward. Unless no issues in testing, I intend to merge this before EOD tomorrow (Thursday) 🤞

I understand the urgency. I'm fine if we fix this as part of a follow up if we're really rushed.

Reply:
Thanks for the feedback and direction. I think the heading needs work before we can effectively discuss reusing it for Login - not in its current form (passed into Step Container with "sub-text" for sub-headings, etc.). I've reworked things and removed the need for a wrapper class.